### PR TITLE
export cookie types

### LINKED ISSUE #213: Google phasing out third-party cookies
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

Hi
Google plans soon to activate phasing out third-party cookies
See Third-Party Cookies doc: https://privacysandbox.com/open-web/#the-privacy-sandbox-timeline

Doc about phasing out third-party cookies: https://developer.chrome.com/en/docs/privacy-sandbox/third-party-cookie-phase-out/#partitioned-cookies

This feature request requires the possibility to add "Partitioned" to the cookie like Secure.

### Motivation

Soon activation from Google
Google plans soon to activate phasing out third-party cookies (Search Third-Party Cookies doc: https://privacysandbox.com/open-web/#the-privacy-sandbox-timeline)

### Example

Extend interface CookieOptions to allow the possibility to set Partioned.

I think that request some dependency on https://github.com/jshttp/cookie

### PR DESCRIPTION
**Pull request by gurgunday**
Not sure if there's a way to this without creating an extra interface

Closes #213

### CODE REVIEW

File: types/types.d.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import type * as Fastify from 'fastify';
 import { FastifyPluginCallback } from 'fastify';
+import { SerializeOptions } from "@fastify/cookie/types/plugin"
```
Comment (mcollina) [inline types/types.d.ts:5]: Please avoid a deep require, all public interfaces should be exported from the entry point.

Comment (gurgunday) [reply types/types.d.ts]: I see, updated the PR

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by gurgunday** [comment]
I initially thought encode wasn’t part of the cookie options so it’s omitted here, I have to change that before merging

**Pull request merged**